exercises/complex-numbers/examples/success-standard/src/ComplexNumbers.hs

```diff
@@ -35,6 +36,9 @@ conjugate (Complex a b) = Complex a (-b)
 abs :: Floating a => Complex a -> a
 abs (Complex a b) = sqrt (exp2 a + exp2 b)
 
+expComplex :: Num a => Complex a -> Complex a
```

Comment:
just as `abs` and `div` use their respective `Prelude` names, I think `exp` should as well, instead of getting a `Complex` suffix. Take a look at the `import Prelude hiding (div, abs)` and adjust accordingly. Or, explain why `exp` should have a different treatment than `abs` and `div`.

Reply:
I agree. However, I'm not experienced enough with Haskell to know how to hide Prelude.exp and still use it (needed for e = `exp 1`). If someone else knows how to do this, I'd love to know and would be happy to remove the suffix.

Reply:
```Haskell
import Prelude hiding (whatEverFunctionYouWantToHide)
```

Reply:
> and still use it

Good point. Try `Prelude.exp`, which should still work even if `hiding` `exp`.

Reply:
I think that should work, ` gchi` session which demonstrates my belief

```
$ ghci                                                           (03-10 10:10)
GHCi, version 8.2.2: http://www.haskell.org/ghc/  :? for help
Prelude> import Prelude hiding (exp)
Prelude> exp

<interactive>:2:1: error: Variable not in scope: exp
Prelude> Prelude.exp 1
2.718281828459045
```